admin/src/containers/Users/Users.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Users from '../../components/Users/Users';
+import config from '../../config';
+import session from '../../helpers/Session';
+
+class UsersContainer extends React.Component {
```

Comment:
do we have any loading state functionality? It would be nice to have some sort of indication of loading for this stuff in state.

Reply:
@Jakedavies we do have a loader and I added it. I forgot about it..